playbooks/openshift-master/private/redeploy-named-certificates.yml

```diff
@@ -0,0 +1,4 @@
+---
+- import_playbook: named-certificates.yml
```

Comment:
I'm not sure I understand the usage of both private/named-certificates.yml and private/redeploy-named-certificates.yml playbooks.  Couldn't the `vars` just be added to the one play from named-certificates.yml and just use private/redeploy-named-certificates.yml?

Also, yamllint issues.

Reply:
Hey,
I was going with the same practice as redeploying master certs playbook. Ok, I understand the point. So you wan't to include directly the tasks in `private/redeploy-named-certificates.yml` - and totally skip the import of `named-certificates.yml`.

I will work on that. Also I will fix the yamllint issues.